package.json

```diff
@@ -1,29 +1,31 @@
 {
   "name": "devicepilot",
   "description": "DevicePilot SDK for IoT Analytics",
-  "version": "1.2.5",
-  "main": "index.js",
+  "version": "2.0.0",
+  "main": "./dist/index.umd.js",
   "repository": "https://github.com/DevicePilot/devicepilot-js.git",
-  "homepage": "https://www.devicepilot.com",
-  "author": "thomasmichaelwallace <tom.wallace@devicepilot.com>",
-  "license": "MIT",
   "scripts": {
+    "prepack": "rollup -c",
     "test": "jest",
     "testing": "jest --watch"
   },
+  "engines": {
+    "node": ">= 7.6.0"
+  },
   "dependencies": {
+    "axios": "^0.19.0",
     "flat": "^4.1.0",
-    "request": "^2.88.0",
-    "request-promise-native": "^1.0.5"
+    "jest": "^24.9.0",
+    "universal-url": "^2.0.0"
   },
   "devDependencies": {
```

Comment:
See my last comment - you want `dependncies` to not include `jest`, and `devDependencies` to read:
```
    "@babel/core": "^7.5.5",
    "@babel/plugin-transform-modules-commonjs": "^7.5.0",
    "eslint": "^6.2.2",
    "eslint-config-airbnb-base": "^14.0.0",
    "eslint-plugin-import": "^2.18.2",
    "jest": "^24.9.0",
    "rollup": "^1.20.3",
```

Your approach makes it so that `jest` becomes a dependency of anyone using our package (it's in `dependencies`), and ignores the `yarn` warning (because @babel/core is requried but not specified).